ignite/pkg/env/env.go

```diff
@@ -0,0 +1,11 @@
+package env
+
+import "os"
+
+const (
+	debug = "IGN_DEBUG"
```

Comment:
```suggestion
	debug = "IGNT_DEBUG"
```

Think this is a bit better